justifi-tech/example-app
# [ENG-1994] Implements Payment List

### Payments List as per Ticket [ENG-1994](https://justifi-ai.atlassian.net/browse/ENG-1994)

- Using fake data
- No Tenant switcher.
- No checking Platform or Seller.

### QA Plan
- go to `/payments`
- check the look and feel as per the mockup in the Ticket.


<img width="2555" alt="Screenshot 2022-11-22 at 15 29 29" src="https://user-images.githubusercontent.com/183364/203393939-14a6c4ce-7a56-4b8e-a8a0-38823c8303b2.png">
